modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/IS/oauth/jwt_token_issuer_addremaininguserattribute.toml

```diff
@@ -1,8 +1,7 @@
 [server]
```

Comment:
rename the file name "addremaininguserattribute.toml" should have a separator

Reply:
Fixed.